File: 2.js
let sentence = prompt(); // ввод строки
let k = 0;
for (let letter = 'a'.charCodeAt(0); letter <= 'z'.charCodeAt(0); letter++) { // перебор алфавита
    if (sentence.includes(String.fromCharCode(letter))) { // поиск букв в строке
        k++; // подсчёт количества
    }
}
console.log(k == 26 ? "true" : "false");

File: 2.cpp
#include <iostream>
#include <string>
#include <cctype>
using namespace std;
int main() {
    int k = 0;
    string sentence;
    getline(cin, sentence); // ввод строки без учёта пробелов
    for (char letter = 'a'; letter <= 'z'; letter++) // перебор букв алфавита
    {
        if (sentence.find(letter) < 1000) { // поиск букв в строке
            k++; // счётчик встречающихся букв
        }
    }
    if (k == 26) {
        cout << "true";
    } else {
        cout << "false";
    }
    return 0;
}
